src/cohort-manager.js

```diff
@@ -0,0 +1,102 @@
+import Cohort from './cohort.js'
+import { Student } from './student.js'
+
+class CohortManager {
+  constructor() {
+    this.cohortList = []
+    this.allStudents = []
+    this.studentIdCount = 1
+  }
+
+  createCohort(name) {
+    if (name.length === 0) throw new Error('Cohort needs a name!')
+    const cohort = new Cohort(name)
+    this.cohortList.push(cohort)
+    return cohort
+  }
+
+  removeCohort(name) {
+    if (name.length === 0) throw new Error('Please enter a cohort name')
+    const foundCohort = this.findCohort(name)
+    return (this.cohortList = this.cohortList.filter(
+      (item) => item.name !== foundCohort.name
+    ))
```

Comment:
Although I appreciate the effort to try and make things concise, this is difficult to read.

I would split this into two separate lines: 

```js
this.cohortList = ...
return this.cohortList
```